packages/page-bounties/src/BountyStatusView.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017-2021 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { DeriveCollectiveProposal } from '@polkadot/api-derive/types';
+import type { BountyStatus } from '@polkadot/types/interfaces';
+
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+
+import { LabelHelp } from '@polkadot/react-components';
+
+import { useTranslation } from './translate';
+import VotingDescription from './VotingDescription';
+
+interface Props {
+  bountyStatus: string;
+  className?: string;
+  proposals?: DeriveCollectiveProposal[];
+  status: BountyStatus;
+}
+
+function BountyStatusView ({ bountyStatus, className = '', proposals, status }: Props): React.ReactElement<Props> {
+  const { t } = useTranslation();
+
+  const statusHelpMessages: Record<string, string> = useMemo(() => ({
```

Comment:
`useRef` is probably more applicable since it will never change. (`useMemo` may be re-done when React feels like it)